src/NzbDrone.Core/Notifications/NotificationService.cs

```diff
@@ -10,13 +10,15 @@
 using NzbDrone.Core.Qualities;
 using NzbDrone.Core.ThingiProvider;
 using NzbDrone.Core.Tv;
+using NzbDrone.Core.Tv.Events;
 
 namespace NzbDrone.Core.Notifications
 {
     public class NotificationService
         : IHandle<EpisodeGrabbedEvent>,
           IHandle<EpisodeImportedEvent>,
           IHandle<SeriesRenamedEvent>,
+          IHandle<EpisodeFileDeletedEvent>,
```

Comment:
`SeriesDeletedEvent` needs to be added as well.

I think both of these should be handled asynchronously as well (like processed and rename events).

Reply:
added `SeriesDeletedEvent`

Also updated to include async